docs/UserGuide.md

```diff
@@ -141,6 +141,19 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Locating a person by tag: `findtag`
+
+Finds all persons whose tags contain the specified text (case-insensitive) and displays them in the list.
+
+Format: `findtag TAG`
```

Comment:
not sure if commands are required to be in camelcase, you can just resolve this comment.